src/components/Profile/UnverifiedProfile.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
```

Comment:
`I don't see `Component` used in this file, so:
```suggestion
import React from 'react';
```